skrub/_dataframe/_polars.py

```diff
@@ -263,5 +280,47 @@ def split_num_categ_cols(table):
     return num_cols, categ_cols
 
 
+def _check_selector(columns):
+    if not isinstance(columns, Selector):
+        return columns
+    if columns is Selector.ALL:
+        return cs.all()
+    elif columns is Selector.NONE:
+        return []
+    elif columns is Selector.NUMERIC:
+        return cs.numeric()
+    elif columns is Selector.CATEGORICAL:
+        return cs.string(include_categorical=True)
+    # we have covered all items in the enumeration
+    assert False
+
+
 def select(dataframe, columns):
-    return dataframe.select(columns)
+    return dataframe.select(_check_selector(columns))
```

Comment:
this helper wouldn't really be appropriate in pandas because in pandas depending on the selector we need to select with either `[]` or `select_dtypes`.
in polars we can just `select()` the output of `_check_selector`